src/banana.js

```diff
@@ -180,6 +204,40 @@ module.exports = function bananaChecker( dir, options, logErr ) {
 		} );
 	}
 
+	let sourceMessageLeadingWhitespace = [];
+	let documentationMessageLeadingWhitespace = [];
+	if ( !options.allowLeadingWhitespace ) {
+		sourceMessageLeadingWhitespace = sourceMessageKeys.filter(
+			function ( message ) {
+				return sourceMessages[ message ].match( leadingWhitespaceRegex );
```

Comment:
Prefer regex.test() which returns a boolean.

Reply:
We consistently use .match() in this codebase currently. I'd rather not make that change just here, if that's OK?